docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -278,6 +278,23 @@
     "estimator.options.resilience.pec.max_overhead = 100"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "a6c87cd7",
+   "metadata": {},
+   "source": [
+    "## When to use each method\n",
+    "\n",
+    "| Method | When to use | Notes|\n",
+    "| --- | --- | --- |\n",
+    "| Dynamical decoupling | Use with circuits in which one or more qubits have long idle times | Circuits with dense two-qubit gates structures (for example, see the paper [Evidence for the utility of quantum computing before fault tolerance](https://doi.org/10.1038/s41586-023-06096-3)). Also, Heron devices are less susceptible to the dominating noise that is suppressed by simple DD sequences (cross-talk) |\n",
+    "| Pauli twirling | Always use | There is a trade-off with the execution time (devices execute more shots faster than more circuits); therefore, using Pauli twirling with enough randomization (even when lowering the shots to keep the total number of shots constant) causes a slowdown |\n",
```

Comment:
> Always use

That kind of contradicts with the note that says there is a trade off. E.g. it's really "always use if you can afford it" lol. How do users know whether to make that trade off?

Reply:
And is gate twirling always necessary if I'm not doing mitigation?